src/components/Scheme/SchemeContainer.tsx

```diff
@@ -125,15 +125,16 @@ class SchemeContainer extends React.Component<IProps, IState> {
     const daoAvatarAddress = daoState.address;
     const schemeState = this.props.data[0];
     const approvedProposals = this.props.data[2];
-    const inInfoTab = this.props.location.pathname.match(/info\/*$/i);
+    const isActive = getSchemeIsActive(schemeState);
+    const isProposalScheme = PROPOSAL_SCHEME_NAMES.includes(schemeState.name);
+    const isBountyScheme = schemeName(schemeState, schemeState.address) === "Standard Bounties";
```

Comment:
how this is related ?

Reply:
it is clarifying the logic related to the change I'm making, in an effort to avoid errors in the future and facilitate understanding the code.

Reply:
 how this specific scheme is related ?

Reply:
I improved the code readability in the flow of logic that is related to the bug fix, added comments, etc.  if I succeeded, then it will be easier to spot problems than it was before, just by eyeballing. The next person who has to come in and fix something in that code, like I did today, will have an easier time than I did